### ISSUE TITLE
Where do you get map names?

### ISSUE THREAD

**Issue by RadNotRed** [Holfz/ApexRPC#11]
Good afternoon, I was looking to contribute and add some of the new maps in rotation for Team DM such as `mp_rr_freedm_map_name_the_core`, and I was confused on where to find the shortened name like `freedm_map_name_skulltown` goes with `skull-town`. I looked at the in rotation maps on the wiki and game files, but `freedm_map_name_skulltown` doesn't exist in files, it comes up as `mp_rr_freedm_skulltown`, which does not work. `parsedLevel` warns for the proper name that it sees and you'd throw that in. 
TLDR: Where do you find these shortened names of the unknown maps logged by `parsedLevel` for images?
sorry if that was confusing, i kind of confused myself writing that 

Also in the files, it shows skulltown as
map_name  "#FREEDM_MAP_NAME_SKULLTOWN" 
image gungame_skull_town
so the part of `"skull-town"` is what I'm looking for

**Comment by RadNotRed** [comment]
im an idiot

**Comment by Holfz** [comment]
Hello, The map name itself is came from what Apex Legends send to Steam. And it will get parsed with two rules

1) If the level (map name) is existed in `Translation` table then it will use it without doing anything
2) If it's not found the level (map name) in `Translation` table, It will try parsing using `Termination` table and try lookup into `Translation` table once again

Hope I understand you correctly.

Regard,
Holfz

**Comment by RadNotRed** [comment]
> Hello, The map name itself is came from what Apex Legends send to Steam. And it will get parsed with two rules
> 
>     1. If the level (map name) is existed in `Translation` table then it will use it without doing anything
> 
>     2. If it's not found the level (map name) in `Translation` table, It will try parsing using `Termination` table and try lookup into `Translation` table once again
> 
> 
> Hope I understand you correctly.
> 
> Regard, Holfz

Yeah that explained it, thank you
Just another question, where do you find your images? I know they go onto the rpc but do you just use a default picture from apex wiki or are they in game files somewhere cropped to 500x500

**Comment by Holfz** [comment]
Hello, I just quickly searched for some good images from the internet and cropped them into 500x500

**Comment by RadNotRed** [comment]
> Hello, I just quickly searched for some good images from the internet and cropped them into 500x500

awesome thanks
